xcm-simulator/src/storagehub/configs/mod.rs

```diff
@@ -564,7 +564,7 @@ parameter_types! {
     pub const ChallengesQueueLength: u32 = 100;
     pub const CheckpointChallengePeriod: u32 = 10;
     pub const ChallengesFee: Balance = 1 * UNIT;
-    pub const StakeToChallengePeriod: Balance = 1_000_000 * UNIT;
+    pub const StakeToChallengePeriod: Balance = 200 * UNIT; // TODO: Change this value into something much higher like 1_000_000 * UNIT
```

Comment:
Is there a reason for this change? Was it generating issues in tests?

Reply:
Yes, the maximum challenge period was too big, compared to the checkpoint challenges. For tests, periods of this magnitude are fine, but for the actual runtime configurations it should be numbers like `1_000_000 * UNIT`. This is handled with the introduction of pallet parameters